mesmer/mesmer_x/train_utils_mesmerx.py

```diff
@@ -358,16 +364,16 @@ def evaluate(self, coefficients_values, inputs_values, forced_shape=None):
         locals = {**coefficients_values, **inputs_values}
 
         # Evaluation 3: parameters
```

Comment:
```suggestion
        # evaluate parameters
```

oh yes I thought that is the "third step" but it was the third evaluation step!